peerconnection.go

```diff
@@ -882,23 +834,19 @@ func (pc *PeerConnection) SetLocalDescription(desc SessionDescription) error {
 		return err
 	}
 
-	// Call the appropriate event handlers to signal that ICE candidate gathering
-	// is complete. In reality it completed a while ago, but triggering these
-	// events helps maintain API compatibility with the JavaScript/Wasm bindings.
-	if err := pc.signalICECandidateGatheringComplete(); err != nil {
-		return err
+	if !pc.iceGatherer.agentIsTrickle {
+		return nil
 	}
-
-	return nil
+	return pc.iceGatherer.Gather()
```

Comment:
Again same deal. This is how it should be but this is a change from original setup. Will this make a performance impact for people?

Reply:
This will only fire for users that are doing `Trickle` and will not block. So shouldn't have an impact!